synthesis/tests/BUILD

```diff
@@ -42,6 +43,16 @@ place_and_route(
     synthesized_rtl = ":verilog_counter_synth",
 )
 
+gds_write(
+	name = "counter_asic",
+	implemented_rtl = ":counter_place_and_route",
+	klayout_lyt = "klayout.lyt",
+	klayout_lef = "klayout_tech.lef",
+	fill_config = "fill.json",
+	additional_lef = ["sky130_fd_sc_hd_merged.lef"],
```

Comment:
can we get those thru dependencies?

Reply:
I'm afraid that if we place all those files under single `dependencies` attribute we would have problems with distinguishing which file is which as they will be stored on a list and we would have to rely on file extensions.

Reply:
see https://github.com/hdl/bazel_rules_hdl/tree/main/dependency_support/com_google_skywater_pdk

Reply:
Oh, I see. I was able to use PDK dependencies to pass down LEF and GDS files through `StandardCellInfo` provider. From now on there is no need to specify `input_lef` and `input_gds` attributes. However I've left those attributes - they might be useful in some custom cases.